code/__HELPERS/mobs.dm

```diff
@@ -284,7 +284,30 @@ Proc for attack log creation, because really why not
 /proc/is_neotheology_disciple(mob/living/L)
 	if(istype(L) && L.get_core_implant(/obj/item/weapon/implant/core_implant/cruciform))
 		return TRUE
+	return FALSE
+
+/proc/is_acolyte(mob/living/L)
+	if(!isliving(L))
+		return FALSE
+	var/obj/item/weapon/implant/core_implant/cruciform/C = L.get_core_implant(/obj/item/weapon/implant/core_implant/cruciform)
+	if(C && C.get_module(CRUCIFORM_PRIEST))
+		return TRUE
+	return FALSE
```

Comment:
Should this be changed to work with https://github.com/discordia-space/CEV-Eris/pull/6233?

Reply:
This PR should be retouched after #6233